Automattic/wp-calypso
# Adminbar: hide in WP Mobile Apps

Related Post p9oQ9f-ow-p2

When request any part of the calypso from the mobile apps (iOS and Android) we want to hide the adminbar both when the user is logged in as well as logged out. 

This done so that the user doesn't see the admin bar navigation inside the app which can cause confusion. Since there is navigation already provided by the Apps themself. 

#### Changes proposed in this Pull Request

* This PR tried to fix the https://github.com/wordpress-mobile/WordPress-Android/issues/13880, https://github.com/wordpress-mobile/WordPress-iOS/issues/7716 issue by making sure that we hide the admin bar view on app views.

#### Testing instructions
* Install an extension to your browser that modifies the headers. 
1. Add a new device in the Resonsive design testing tool. 

<img width="383" alt="Screen Shot 2021-01-25 at 5 19 55 PM" src="https://user-images.githubusercontent.com/115071/105786337-b9ca2100-5f31-11eb-84bd-af1452a6f70f.png">

<img width="807" alt="Screen Shot 2021-01-25 at 5 22 33 PM" src="https://user-images.githubusercontent.com/115071/105786430-ee3ddd00-5f31-11eb-83a6-67be2bcf5e01.png">

the device should have the following agent. 
```
Mozilla/5.0 (Linux; Android 6.0; Android SDK built for x86_64 Build/MASTER; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/44.0.2403.119 Mobile Safari/537.36 wp-android/4.7
```
or 
```
Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/16B91 wp-iphone/12.1
```

* Select the device. Notice that the masterbar as well as the inline help is gone in calypso.
<img width="928" alt="Screen Shot 2021-01-25 at 5 23 10 PM" src="https://user-images.githubusercontent.com/115071/105786486-044b9d80-5f32-11eb-81d5-49f8eca094c1.png">

Page in question that we want to clean up is 
http://calypso.localhost:3000/theme/spearhead/example.com

@tyxla would love your 2 cents on the approach.
